jerry-core/debugger/jerry-debugger.c

```diff
@@ -1 +1,101 @@
-//TODO: Implement the socket communication
+/* Copyright 2016 University of Szeged.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "jerry-debugger.h"
+#include "jerry-port.h"
+
+#define PORT 5001
+#define BLACKLOG 1
+#define BUFFER_SIZE 128
+
+int sock;                     /**< return value of socket(), used other methods */
```

Comment:
the comment is still not clear. Focus on the function of this variable: why is it used for? what is this good for?
For example, "socket file descriptor for the remote communication"